apps/condo/domains/miniapp/access/B2BApp.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Generated by `createschema miniapp.B2BApp 'name:Text;'`
+ */
+
+const { throwAuthenticationError } = require('@condo/domains/common/utils/apolloErrorFormatter')
+
+async function canReadB2BApps ({ authentication: { item: user } }) {
+    if (!user) return throwAuthenticationError()
+    if (user.deletedAt) return false
```

Comment:
Why do we need to check deletedAt? Deleted user cannot be authenticated

Reply:
👍

Reply:
:)

Reply:
But what if I was already logged in, when deletion happens? 🤔